docs/intro.rst

```diff
@@ -298,22 +298,83 @@ StrBool
 
 Float
 .....
+Check if value is a float or can be converted to a float.
+Supports ``lte``, ``gte``, ``lt``, ``gt`` parameters,
```

Comment:
We have a similar thing in documentation with string.

```
The String is base checker in trafaret which just test that value is string. Also String has a lot of helpful modification like Email and Url.

>>> t.String().check('this is my string')
'this is my string'
Options:

allow_blank (boolean) - indicates if string can be blank or not
min_length (integer) - validation for minimum length of receive string
max_length (integer) - validation for maximum length of receive string
The simple examples of usage:

>>> t.String(allow_blank=True).check('')
''
>>> t.String(min_length=1, max_length=10).check('no so long')
'no so long'
```

I think we need follow this style. So, first show how work simple `Float` and after describe options and show how these options work.

What do u think?

Reply:
If u wanna do fixs make this in next pull request.